src/PasswordlessLoginUrlGenerator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Grosv\LaravelPasswordlessLogin;
+
+use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Contracts\Routing\UrlGenerator;
+
+class PasswordlessLoginUrlGenerator
+{
+    /**
+     * @var \Grosv\LaravelPasswordlessLogin\PasswordlessLoginOptions
+     */
+    protected $options;
+
+    /**
+     * @var \Illuminate\Contracts\Routing\UrlGenerator|\Illuminate\Routing\UrlGenerator
+     */
+    protected $url;
+
+    /**
+     * PasswordlessLoginManager constructor.
+     *
+     * @param \Grosv\LaravelPasswordlessLogin\PasswordlessLoginOptions $options
+     * @param \Illuminate\Contracts\Routing\UrlGenerator               $url
+     */
+    public function __construct(PasswordlessLoginOptions $options, UrlGenerator $url)
+    {
+        $this->options = $options;
+        $this->url = $url;
+    }
+
+    /**
+     * @param \Illuminate\Contracts\Auth\Authenticatable $user
+     *
+     * @return string
+     */
+    public function generate(Authenticatable $user)
+    {
+        return $this->url->temporarySignedRoute(
+            $this->options->loginRouteName(),
+            $this->options->loginRouteExpires() * 60,
```

Comment:
Note that we'd better require the value **in seconds**, not in minutes. Because Laravel had a breaking change in 5.8 that changes cache TTL unit from minutes to seconds. We should follow the philosophy.

- https://laravel.com/docs/5.8/upgrade#cache